src/index.scss

```diff
@@ -0,0 +1,141 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+  background-image: url(star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 .stars__star:nth-of-type(1),
+.stars--2 .stars__star:nth-of-type(-n+2),
+.stars--3 .stars__star:nth-of-type(-n+3),
+.stars--4 .stars__star:nth-of-type(-n+4),
+.stars--5 .stars__star:nth-of-type(-n+5) {
+  background-image: url(star-active.svg);
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  left: 1px;
+  top: 1px;
+  border-radius: 5px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  font-size: 10px;
+  line-height: 14px;
+  padding: 32px 16px 16px;
+  color: #060b35;
+}
+
+.card__image {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.card__image-img {
+  height: 134px;
+  width: 160px;
+}
+
+.card__product-name {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
```

Comment:
pick the main font styles and set them on body and only change them in places you need